core/designsystem/src/main/java/org/mifos/mobilewallet/mifospay/designsystem/component/MifosUserImage.kt

```diff
@@ -0,0 +1,40 @@
+package org.mifos.mobilewallet.mifospay.designsystem.component
+
+import android.graphics.Bitmap
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.layout.ContentScale
+
+/**
+ * @author pratyush
+ * @since 20/12/2023
+ */
+
+@Composable
+fun MifosUserImage(
+    bitmap: Bitmap?,
+    modifier: Modifier = Modifier,
+    username: String? = null
+) {
+    if (bitmap == null) {
+        MifosTextUserImage(
+            modifier = modifier,
+            text = username?.firstOrNull()?.toString() ?: "J"
```

Comment:
Here image should have 3 condition
1. If bitmap is not null then load bitmap 
2. If bitmap is null then check username or name and set first char
3. As bitmap is null and we set username or name first char then we need to set plus circular image at bottom right of the image that allows user to open update user profile photo.

Reply:
The 3rd condition can be handled after the user clicks on the `Edit Profile` button @therajanmaurya . Hence Adding a plus icon in my opinion isn't the best alternative